completion/available/docker-machine.completion.bash

```diff
@@ -1,431 +1,3 @@
-    fi
-}
-
-_docker_machine_stop() {
-    if [[ "${cur}" == -* ]]; then
-        COMPREPLY=($(compgen -W "--help" -- "${cur}"))
-    else
-        COMPREPLY=($(compgen -W "$(_docker_machine_machines --filter state=Running)" -- "${cur}"))
-    fi
-}
-
-_docker_machine_upgrade() {
-    if [[ "${cur}" == -* ]]; then
-        COMPREPLY=($(compgen -W "--help" -- "${cur}"))
-    else
-        COMPREPLY=($(compgen -W "$(_docker_machine_machines)" -- "${cur}"))
-    fi
-}
-
-_docker_machine_url() {
-    if [[ "${cur}" == -* ]]; then
-        COMPREPLY=($(compgen -W "--help" -- "${cur}"))
-    else
-        COMPREPLY=($(compgen -W "$(_docker_machine_machines)" -- "${cur}"))
-    fi
-}
-
-_docker_machine_version() {
-    if [[ "${cur}" == -* ]]; then
-        COMPREPLY=($(compgen -W "--help" -- "${cur}"))
-    else
-        COMPREPLY=($(compgen -W "$(_docker_machine_machines)" -- "${cur}"))
-    fi
-}
-
-_docker_machine_help() {
-    if [[ "${cur}" == -* ]]; then
-        COMPREPLY=($(compgen -W "--help" -- "${cur}"))
-    else
-        COMPREPLY=($(compgen -W "${commands[*]}" -- "${cur}"))
-    fi
-}
-
-_docker_machine_docker_machine() {
-    if [[ " ${wants_file[*]} " =~ " ${prev} " ]]; then
-        _filedir
-    elif [[ " ${wants_dir[*]} " =~ " ${prev} " ]]; then
-        _filedir -d
-    elif [[ "${cur}" == -* ]]; then
-        COMPREPLY=($(compgen -W "${flags[*]} ${wants_dir[*]} ${wants_file[*]}" -- "${cur}"))
-    else
-        COMPREPLY=($(compgen -W "${commands[*]}" -- "${cur}"))
-    fi
-}
-
-_docker_machine() {
-    COMPREPLY=()
-    local commands=(active config create env inspect ip kill ls mount provision regenerate-certs restart rm ssh scp start status stop upgrade url version help)
-
-    local flags=(--debug --native-ssh --github-api-token --bugsnag-api-token --help --version)
-    local wants_dir=(--storage-path)
-    local wants_file=(--tls-ca-cert --tls-ca-key --tls-client-cert --tls-client-key)
-
-    # Add the use subcommand, if we have an alias loaded
-    if [[ ${DOCKER_MACHINE_WRAPPED} = true ]]; then
-        commands=("${commands[@]}" use)
-    fi
-
-    local cur prev words cword
-    _get_comp_words_by_ref -n : cur prev words cword
-    local i
-    local command=docker-machine command_pos=0
-
-    for (( i=1; i < ${cword}; ++i)); do
-        local word=${words[i]}
-        if [[ " ${wants_file[*]} ${wants_dir[*]} " =~ " ${word} " ]]; then
-            # skip the next option
-            (( ++i ))
-        elif [[ " ${commands[*]} " =~ " ${word} " ]]; then
-            command=${word}
-            command_pos=$i
-        fi
-    done
-
-    local completion_func=_docker_machine_"${command//-/_}"
-    if declare -F "${completion_func}" > /dev/null; then
-        ${completion_func}
-    fi
-
-    return 0
-}
-
-complete -F _docker_machine docker-machine docker-machine.exe
+# shellcheck shell=bash
+_log_error '"docker-machine" is now deprecated, and as such the bash completion for it is also deprecated.
```

Comment:
Don't know if we discussed this before, but do we want to raise the deprecation as an error? Wouldn't a warning be sufficient?

Reply:
Hmm, I am unsure. I think errors are things that should be reported to us, while warning are warnings to the user, so I guess this really should just be log_warning. I say we merge this, and I will open an issue about it

Reply:
I think we both agree that it should be a warning then?

Reply:
I updated the PR with a commit that changes all places like this